### PR TITLE
Fix partners logos

### PR DESCRIPTION
**Pull request by MatteoRobbiati**
In this PR:

- logos are now vertically centered;
- logos has their official colors;
- logos' section is put on top (after qibo's description);
- partners title is now bigger;

TO DO:

- [x] hyper-links to partners' sites.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -185,6 +185,50 @@ export default function Home() {
...
+              </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://www.quantumlah.org/"> 
+                <img src='/images/home/cqt.png' alt='/'/>
+              </a>
+            </div>
+          </div>
+        </div>
+      </Hero3>
```
Comment (AleCandido) [inline pages/index.js:230]: I would limit the `Hero` to a single one, and the rest of the page to a more ordinary flow.
I wonder if using a `Hero` at all is really useful. However, for this round fine as is.

**Review by AleCandido — APPROVED** [review]

File: pages/index.js (hunk 1 above)
Comment (MatteoRobbiati) [reply pages/index.js:230]: Thank you. I open an issue for this.

File: pages/index.js [hunk 2]
```diff
@@ -185,6 +185,50 @@ export default function Home() {
...
+                  <img src='/images/home/infn.png' alt='/'/> 
+                </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://home.cern/"> 
+                <img src='/images/home/cern.png' alt='/'/> 
+              </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://www.nvidia.com/en-usqilimanjaro
```
Comment (andrea-pasquale) [inline pages/index.js:208]: This link is not working

**Review by andrea-pasquale — COMMENTED** [review]
Thanks I really like this version of the logos.

File: pages/index.js (hunk 2 above)
Comment (MatteoRobbiati) [reply pages/index.js]: Fixed. Thank you.

**Pull request merged**